matrix_client/client.py

```diff
@@ -379,3 +379,27 @@ def get_user(self, user_id):
         """
 
         return User(self.api, user_id)
+
+    def get_public_rooms_ids(self):
```

Comment:
I'd be happy to. If you could move the alias patches to a separate PR so I can review and merge them separate from this discussion I'd be most grateful :)